static/app/views/pipeline/navigation.tsx

```diff
@@ -0,0 +1,43 @@
+import {Fragment} from 'react';
+
+import {NAV_GROUP_LABELS} from 'sentry/components/nav/constants';
+import {SecondaryNav} from 'sentry/components/nav/secondary';
+import {PrimaryNavGroup} from 'sentry/components/nav/types';
+import useOrganization from 'sentry/utils/useOrganization';
+import {makePipelinePathname} from 'sentry/views/pipeline/pathnames';
+import {COVERAGE_BASE_URL, TESTS_BASE_URL} from 'sentry/views/pipeline/settings';
+
+type PipelineSecondaryNavProps = {
+  children: React.ReactNode;
+};
+
+function PipelineSecondaryNav({children}: PipelineSecondaryNavProps) {
+  const organization = useOrganization();
+  const coveragePathname = makePipelinePathname({
+    organization,
+    path: `/${COVERAGE_BASE_URL}/`,
+  });
+  const testsPathname = makePipelinePathname({
+    organization,
+    path: `/${TESTS_BASE_URL}/`,
+  });
+
+  return (
+    <Fragment>
+      <SecondaryNav group={PrimaryNavGroup.PIPELINE}>
+        <SecondaryNav.Header>
+          {NAV_GROUP_LABELS[PrimaryNavGroup.PIPELINE]}
+        </SecondaryNav.Header>
+        <SecondaryNav.Body>
+          <SecondaryNav.Section>
+            <SecondaryNav.Item to={coveragePathname}>Coverage</SecondaryNav.Item>
```

Comment:
These should be wrapped in the translation utils - `t('Coverage')` and `t('Tests')`

Reply:
Whoops, gotta get used to that 😅